polly/test/UnitIsl/lit.cfg

```diff
@@ -16,8 +16,15 @@ config.name = 'Polly - isl unit tests'
 #
 # For now we require '&&' between commands, until they get globally killed and
 # the test runner updated.
-execute_external = platform.system() != 'Windows'
-config.test_format = lit.formats.ShTest(execute_external)
+#
+# We prefer the lit internal shell which provides a better user experience on failures
+# unless the user explicitly disables it with LIT_USE_INTERNAL_SHELL=0 env var.
+use_lit_shell = True
+lit_shell_env = os.environ.get("LIT_USE_INTERNAL_SHELL")
+if lit_shell_env:
+  use_lit_shell = not lit.util.pythonize_bool(lit_shell_env)
+
+config.test_format = lit.formats.ShTest(execute_external=not use_lit_shell)
```

Comment:
Do we need to do anything to add or remove the "shell" feature from the config? This is handled in other suites by the llvm_config module:
https://github.com/llvm/llvm-project/blob/main/llvm/utils/lit/lit/llvm/config.py#L75

Reply:
None of the polly tests seem to require "shell", but many of the other test suites do. Right now we aren't able to completely remove `REQUIRES: shell` from all test files, and I'm not sure if there is a way to configure this just for polly without affecting the other suites. 

Not sure if that answered your question, so please let me know if you need further elaboration.

Reply:
If there are no uses of REQUIRES: shell in Polly, then I suppose it's not worth tracking the feature.